tests/std/tests/Dev11_0000000_include_each_header_alone/test.cpp

```diff
@@ -1,6 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#ifndef MEOW_DONT_FAKE_STD
+
+#include <vcruntime_new.h>

```

Comment:
This unfortunately damages the original purpose of this test, which is to guarantee that each header can be included with no prerequisites. I'd prefer to see a separate/new test. Also, Stephan mentioned that there doesn't seem to be any advantage to doing this with each header individually and we could just as well make it an "include all the headers at once" test with `__msvc_all_public_headers.hpp`.

Reply:
If you can fix `<runtime_new>`, we can combine. Advantage is maintaining full list in one place

Reply:
Ah, you already have `__msvc_all_public_headers.hpp`

Reply:
Reverted here, added new implementation.